src/components/Dashboard/TodoList.vue

```diff
@@ -253,6 +254,16 @@
                   <span v-if="inProcessFiling === item.filingId">PROCESSING...</span>
                   <span v-else>PAID</span>
                 </div>
+
+                <!-- change-requested filing -->
+                <div
+                  v-else-if="EnumUtilities.isStatusChangeRequested(item)"
+                  class="todo-subtitle"
+                >
+                  <span>CHANGE REQUESTED</span>
+                  <span class="vert-pipe" />
+                  <span>PAID (filed by {{ item.submitter }} on <DateTooltip :date="item.submittedDate" />)</span>
+                </div>
```

Comment:
There's only 1 type of filing that can be "change requested" right now, but I wrote this generically in case we get others.